lib/cordial/orders.rb

```diff
@@ -36,36 +36,21 @@ def self.find(id:)
     end
 
     # Create a new order.
-    # Posting more than one time the same "orderID" name will generate an error.
+    #
+    # This endpoint does not support the idea of an upsert like others do.
+    # Subsequent calls will fail.
     #
     # @example Usage.
-    # Cordial::Orders.create(
-    #   id: 1,
-    #   email: 'cordial@example.com',
-    #   link_id: '123456',
-    #   mc_id: '645:5b6a1f26e1b829b63c2a7946:ot:5aea409bbb3dc2f9bc27158f:1',
-    #   purchase_date: '2015-01-09 17:47:43',
-    #   items: [{productID: '1',
-    #            sku: '123',
-    #            name: 'Test product',
-    #            attr: { color: 'blue', size: 'L' }}])
-    # @example response whe the orderID is not on cordial
+    # Cordial::Orders.create({...})
```

Comment:
I think we need to be more explicit in the example

Reply:
Since we're going to delegate what the options are to the special Order class, I'd rather people refer to that to see what to pass in rather than the docs here I don't want them to get out of date.